test.js

```diff
@@ -1 +1,11 @@
-console.log('Hello World')
+console.log('Hello World');
+
+console.log('Hello Universe');
+
+console.log('Hello other stuff');
+
+if(foo === "bar") {
```

Comment:
Codacy has a fix for the issue: [Insert `·`](https://app.dev.codacy.org/gh/troubleshoot-codacy/no-suggestions-2/pullRequest?prid=30889)
```suggestion
if (foo === "bar") {
```